### DISCUSSION TITLE
[Open] Deepin-printer installs on Arch Linux Rolling V20.1, but does not work

### DISCUSSION THREAD

**Discussion by serial02** [linuxdeepin/developer-center#3031] [Q&A | 问答板块] [answered]
Hello Deepin Printer Team

I installed the deepin-printer on Arch Linux Rolling V20.1, as well as its dependencies, but it doesn't work.

![Captura de Tela_Opera_20210104164505](https://user-images.githubusercontent.com/32621790/103574910-8acf0b00-4eaf-11eb-930e-09857c2438b4.png)

When opening the printer it appears the information that the CUPS Server is not running and cannot manage the printers. How do I solve this?

![Captura de Tela_dde-printer_20210104164052](https://user-images.githubusercontent.com/32621790/103575022-beaa3080-4eaf-11eb-8db8-266aac347fed.png)

The deepin-printer settings option does not open.

By clicking on the CUPS manager, we are taken to the site: http: // localhost: 631 /
How should I proceed to finish configuring my printer?

Do I need to install anything else?
Note: In Deepin V20.1 1010 it works normally (!)

**Comment by NuLogicSystems** [comment, marked as answer]
Cups isn't running or enabled. 
```
sudo systemctl enable cups.service
sudo systemctl start cups.service
```
You could alternatively use socket based:
`sudo systemctl enable cups.socket`

https://wiki.archlinux.org/index.php/CUPS#Installation
